src/tests/Logary.Targets.Serilog.Tests/Program.fs

```diff
@@ -0,0 +1,145 @@
+﻿module Logary.Targets.Serilog.Tests
+
+open System
+open System.Net
+open Expecto
+open Hopac
+open Logary.Targets
+open Logary.Configuration
+open Logary
+
+module Target =
```

Comment:
Just noting that this is common in *all* targets and could probably be moved to a `Logary.Testing` library that every test lib can then reference.